### PR TITLE
Adding include dirs for utilities to exported AIGS_INCLUDE_DIRS variable

### PR DESCRIPTION
**Pull request by adamrankin**
This enables external projects to use AIGS without knowing the infrastructure of the project

### CODE REVIEW

**Comment by adamrankin** [comment]
Hi David,

I was trying to use your fork of AIGS in a project here and it couldn't find ndicapi.h

I added the location of the utilities to the exported AIGS_INCLUDE_DIRS so that the AIGS_USE_FILE or anyone including the variable will be able to access those headers.

Thoughts?

File: CMakeLists.txt [hunk 1]
```diff
@@ -102,18 +102,23 @@ IF (AIGS_USE_TRACKING)
...
   IF (AIGS_USE_LOGITECH)
+    SET(AIGS_INCLUDE_DIRS ${AIGS_INCLUDE_DIRS} ${CMAKE_SOURCE_DIR}/Utilities/logitech3d)
     SET(AIGS_LIBRARIES ${AIGS_LIBRARIES} vtklogitech3d)
   ENDIF (AIGS_USE_LOGITECH)
   IF (AIGS_USE_POLHEMUS)
+    SET(AIGS_INCLUDE_DIRS ${AIGS_INCLUDE_DIRS} ${CMAKE_SOURCE_DIR}/Utilities/polhemus)
     SET(AIGS_LIBRARIES ${AIGS_LIBRARIES} vtkpolhemus)
   ENDIF (AIGS_USE_POLHEMUS)
   IF (AIGS_USE_MICRONTRACKER)
+    SET(AIGS_INCLUDE_DIRS ${AIGS_INCLUDE_DIRS} ${CMAKE_SOURCE_DIR}/Utilities/polhemus/MicronTrackerFiles)
```
Comment (dgobbi) [inline CMakeLists.txt]: Typo here: the 'polhemus' should be removed from 'polhemus/MicronTrackerFiles'.

**Comment by dgobbi** [comment]
Hi Adam, the change looks good except for the 'polhemus/MicronTrackerFiles' typo.

Most of this code has been merged into PLUS (and Andras does better job of maintaining PLUS than I've been doing for AIGS!)

But I'm still glad to receive any comments and fixes.

**Comment by adamrankin** [comment]
I agree, this is mostly because the project already used AIGS and it was way easier for me to add these includes than to change the project over to PLUS.

Typo has been fixed.

**Comment by dgobbi** [comment]
Thanks, I've squashed the commits and merged into master.

**Pull request closed without merging**